kpconv_torch/io/xyz.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+
+def read_xyz(filepath):
+    """Takes a file path pointing on a 3D point .xyz (text) file and returns the points,
+    the associated colors and the associated classes.
+
+    Parameters
+    ----------
+    filepath: path to a 3D points file with .xyz
+
+    Returns
+    -------
+    points: 2D np.array with type float32
+    colors: 2D np.array with type uint8
+    labels: 1D np.array with type int32
+    """
+    data = np.loadtxt(filepath, delimiter=" ")
+    points = data[:, :3].astype(np.float32)
+    colors = data[:, 3:6].astype(np.uint8)
+    labels = np.squeeze(data[:, 6:7]).astype(np.int32)
```

Comment:
What if there is no classification data in the `.xyz` file?

Reply:
Added.